lib/services/stripe/checkout.ts

```diff
@@ -9,17 +11,27 @@ export const createCheckoutSession = async ({
   baseUrl: string;
   priceId: string;
 }) => {
-  const session = await stripe.checkout.sessions.create({
+  const user = await getUser();
+  const accountId = await getAccountId();
+
+  const checkoutSession = await stripe.checkout.sessions.create({
     cancel_url: `${baseUrl}?status=cancel`,
+    customer_email: user.email,
     line_items: [
       {
         price: priceId,
         quantity: 1,
       },
     ],
+    metadata: {
+      accountId,
+      userId: user.id,
```

Comment:
not sure if we need both, I can just pass `userId` then fetch account from user in the webhook idk.

Reply:
either is fine. Also, maybe, we don't need any info from account table at all.

Reply:
I saw that with the next PR we need it.

Reply:
It would be better if we don't query and add account here but I think it's a very minor improvement. We can keep it like this